src/SFA.DAS.AssessorService.Application.Api.UnitTests/Validators/ExternalApi/Epas/BatchEpaRequestValidator/WhenEpaDateInFuture.cs

```diff
@@ -0,0 +1,44 @@
+﻿using FizzWare.NBuilder;
+using FluentAssertions;
+using FluentValidation.Results;
+using NUnit.Framework;
+using SFA.DAS.AssessorService.Api.Types.Models.ExternalApi.Epas;
+using SFA.DAS.AssessorService.Domain.JsonData;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AssessorService.Application.Api.UnitTests.Validators.ExternalApi.Epas.BatchEpaRequestValidator
+{
+    public class WhenEpaDateInFuture : BatchEpaRequestValidatorTestBase
+    {
+        private ValidationResult _validationResult;
+
+        [SetUp]
+        public async Task Arrange()
+        {
+            var epas = Builder<EpaRecord>.CreateListOfSize(1).All()
+                .With(i => i.EpaDate = DateTime.UtcNow.AddHours(1))
```

Comment:
At last something to comment on....  I'm assuming the External API has been written with incoming times as UTC - and that the docs state that - then if a created record is shown in the UI it would need converting back to local time I'm thinking this is bound to be wrong somewhere perhaps a Tech Debt to look into the whole issue?

Reply:
There is a DateTimeUtils.cs class which does all of this UTC / TimeZone shuffling

The External API makes use of the Json DateTime type which allows you to specify the timezone

Reply:
There needs to be a ticket to make sure this has been consistently done though - a quick check shows lots of examples of it being converting from utc to local, but I didn't know it existed so I'm guessing there are lots of places it isn't being done or I would have stumbled over it by now...